dotnet/test/Azure.Iot.Operations.Services.IntegrationTests/resources/coap-asset.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
```

Comment:
can you move these to the /eng folder so we can use them for other language tests too?